tubesize/src/components/options/apiFallbackSetting.tsx

```
@@ -0,0 +1,52 @@
+                <div className="info-tooltip">
+                    <button
+                        type="button"
+                        className="info-tooltip-trigger"
+                        aria-label="More information about API fallback"
+                    ></button>
+                    <div className="info-tooltip-content" role="tooltip">
+                        Uses our backup server when local extraction fails. It can be slower. See
+                        <a
+                            className="info-tooltip-link"
+                            href="https://github.com/MohamedSayed0573/TubeSize_Extension/blob/main/PRIVACY.md"
+                            target="_blank"
```

Biome (2.4.7)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

(lint/security/noBlankTarget)